decred/tests/benchmark/util/test_database_benchmark.py

```diff
@@ -0,0 +1,48 @@
+"""
+Copyright (c) 2019, the Decred developers
+See LICENSE for details
+"""
+
+import pytest
+import os.path
+from tempfile import TemporaryDirectory
+import time
+
+from decred.util import database
+
+
+def test_benchmark():
```

Comment:
@teknico I wanted to move this from the unit tests, but didn't think it belonged in **integration**. How do you feel about this **benchmark** directory?

Reply:
@buck54321, it looks good to me!